en/advanced_config/bootloader_update.md

```diff
@@ -96,18 +98,24 @@ To update the bootloader:
    In this case you will not be able to install FMUv3 hardware.
    :::
 
-## Dronecode Probe Bootloader Update
+## Debug Probe Bootloader Update
```

Comment:
@mrpollo  Is this correct and OK to change

Reply:
This is _good_ to change, provided the instructions worth with all the linked probes. I will assume so.